### PR TITLE
WheelPicker - add driver

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
WheelPicker - add driver

## Changelog
WheelPicker - add driver
SectionsWheelPicker - add driver

### CODE REVIEW

**Comment by ethanshar** [comment]
@Inbal-Tish Is this ready for review?

**Comment by Inbal-Tish** [comment]
It's ready but incomplete. It doesn't test much currently

File: src/components/WheelPicker/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      expect(driver.getLabel()).toEqual('item #0');
+    });
+
+    it('should get first item\'s text style when no active/inactive colors', () => {
+      const renderTree = render(<TestCase textStyle={{color: Colors.green30}}/>);
+      const index = 0;
+      const driver = WheelPickerItemDriver({renderTree, testID: `${index}`});
+      expect(driver.getLabelStyle()?.color).toEqual(Colors.green30);
+    });
+
```
Comment (adids1221) [inline src/components/WheelPicker/__tests__/index.spec.tsx:85]:
```suggestion
//TODO: Fix these test's using AnimatedStyle mocking
```

File: src/components/WheelPicker/__tests__/index.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      expect(driver.getLabel()).toEqual('item #0');
+    });
+
+    it('should get first item\'s text style when no active/inactive colors', () => {
+      const renderTree = render(<TestCase textStyle={{color: Colors.green30}}/>);
+      const index = 0;
+      const driver = WheelPickerItemDriver({renderTree, testID: `${index}`});
+      expect(driver.getLabelStyle()?.color).toEqual(Colors.green30);
+    });
+//TODO: Fix these test's using AnimatedStyle mocking
```
Comment (Inbal-Tish) [inline src/components/WheelPicker/__tests__/index.spec.tsx:85]:
```suggestion

        //TODO: Fix these test's using AnimatedStyle mocking
```

File: src/components/WheelPicker/__tests__/index.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    });
+
+    it('should get first item\'s text style when no active/inactive colors', () => {
+      const renderTree = render(<TestCase textStyle={{color: Colors.green30}}/>);
+      const index = 0;
+      const driver = WheelPickerItemDriver({renderTree, testID: `${index}`});
+      expect(driver.getLabelStyle()?.color).toEqual(Colors.green30);
+    });
+
+        //TODO: Fix these test's using AnimatedStyle mocking
```
Comment (adids1221) [inline src/components/WheelPicker/__tests__/index.spec.tsx:86]:
```suggestion
    //TODO: Fix these test's using AnimatedStyle mocking
```

File: src/components/sectionsWheelPicker/mockSections.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+const onDaysChange = jest.fn();
+const onHoursChange = jest.fn();
+const onMinutesChange = jest.fn();
+
+export const sections = [
+  {
+    items: getItems(DAYS),
+    onChange: onDaysChange,
+    // initialValue: selectedDays,
```
Comment (adids1221) [inline src/components/sectionsWheelPicker/mockSections.ts:21]:
```suggestion
```

File: src/components/sectionsWheelPicker/mockSections.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    label: labels[0]
+  },
+  {
+    items: getItems(HOURS),
+    onChange: onHoursChange,
+    // initialValue: selectedHours,
```
Comment (adids1221) [inline src/components/sectionsWheelPicker/mockSections.ts:27]:
```suggestion
```

File: src/components/sectionsWheelPicker/mockSections.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    label: labels[1]
+  },
+  {
+    items: getItems(MINUTES),
+    onChange: onMinutesChange,
+    // initialValue: selectedMinutes,
```
Comment (adids1221) [inline src/components/sectionsWheelPicker/mockSections.ts:33]:
```suggestion
```

**Pull request merged**